apidef/oas/default.go

```diff
@@ -120,6 +127,29 @@ func (s *OAS) BuildDefaultTykExtension(overRideValues TykExtensionConfigParams,
 	return nil
 }
 
+func generateUrlUsingDefaultVariableValues(s *OAS, upstreamURL string) (string, error) {
+	for name, variable := range s.Servers[0].Variables {
+		if strings.Contains(upstreamURL, "{"+name+"}") {
+			if variable.Default == "" {
+				return "", errors.New("server variable " + name + " does not have a default value")
+			}
+			upstreamURL = replaceParameterWithValue(upstreamURL, name, variable.Default)
+		}
+	}
+	if isURLParametrized(upstreamURL) {
+		return "", errors.New("server URL still contains unresolved variables")
```

Comment:
```suggestion
		return "", errors.New("server URL contains undefined variables")
```
the message `still` doesn't make sense from a user POV